conf/application.conf

```diff
@@ -186,7 +194,7 @@ metrics {
 }
 
 # the value below is valid for local environment only
-mongo-encryption.key = "MTIzNDU2Nzg5MDEyMzQ1Ng=="
+json.encryption.key = "MTIzNDU2Nzg5MDEyMzQ1Ng=="
```

Comment:
this needs to be proven mid flight in QA (deploy and make sure journey doesnt break) (make sure PR is raised to be json.encryption.key) also staging dev and prod!    base?